TA-microsoft-graph-security-score/app.manifest

```diff
@@ -1,62 +1,59 @@
 {
-  "schemaVersion": "2.0.0",
-  "info": {
-    "title": "Microsoft Graph Security Score Add-on",
-    "id": {
-      "group": null,
-      "name": "TA-microsoft-graph-security-score",
-      "version": "1.1.1"
+    "dependencies": {},
+    "incompatibleApps": {},
+    "info": {
+        "author": [
+            {
+                "company": null,
+                "email": null,
+                "name": "CrossRealms International"
+            }
+        ],
+        "classification": {
+            "categories": [],
+            "developmentStatus": null,
+            "intendedAudience": null
+        },
+        "commonInformationModels": null,
+        "description": "This Add-on pulls secure score from Microsoft Graph Security API.",
+        "id": {
+            "group": null,
+            "name": "TA-microsoft-graph-security-score",
+            "version": "1.2.0"
+        },
+        "license": {
+            "name": null,
+            "text": null,
+            "uri": null
+        },
+        "privacyPolicy": {
+            "name": null,
+            "text": null,
+            "uri": null
+        },
+        "releaseDate": null,
+        "releaseNotes": {
+            "name": null,
+            "text": null,
+            "uri": null
+        },
+        "title": "Microsoft Graph Security Score Add-on"
     },
-    "author": [
-      {
-        "name": "CrossRealms International",
-        "email": null,
-        "company": null
-      }
-    ],
-    "releaseDate": null,
-    "description": "This Add-on pulls secure score from Microsoft Graph Security API.",
-    "classification": {
-      "intendedAudience": null,
-      "categories": [],
-      "developmentStatus": null
-    },
-    "commonInformationModels": null,
-    "license": {
-      "name": null,
-      "text": null,
-      "uri": null
-    },
-    "privacyPolicy": {
-      "name": null,
-      "text": null,
-      "uri": null
+    "inputGroups": {},
+    "platformRequirements": {
+        "splunk": {
+            "Enterprise": "*"
```

Comment:
Shall we include Cloud here as well?

Reply:
It is fine. Maxmind app has same config.